qep-347-welcome-page-revamp.md

```diff
@@ -0,0 +1,79 @@
+# QGIS Enhancement: Welcome Page Revamp
+
+**Date** 2025/11/11
+
+**Author** Mathieu Pellerin (@nirvn)
+
+**Contact** mathieu@opengis.ch
+
+**Version** QGIS 4.0.0+
+
+## Summary
+
+The current welcome page users land on when opening QGIS has a few
+shortcomings. Here’s a non-exhaustive list of issues:
+
+- The templates section – rarely used - takes too much space when
+weighted against other components, especially the news section.
+- The overall look lacks ties to the QGIS branding and feels outdated.
+- The layout lacks outbound links to QGIS.org and other useful links
+such as the visual release changelog.
+
+Crucially, the current welcome page does not facilitate able and willing
+users into supporting the QGIS project. Considering the crucial role
+donations play in the continuity of QGIS, this feels like a significant
+gap worthy of addressing.
+
+
+## Proposed Solution
+
+This QEP proposes to re-do the welcome page, and to do so in Qt’s Quick
+framework to gain the ability to increase the visual appeal for this
+crucial landing page with new and seasoned users. Embracing the Qt Quick
+framework will allow us to build a modern welcome page based on a
+dynamic GUI with fluid animations and responsive graphics.
+
+Adopting Qt Quick allows us to build much of the welcome page using the
+QML declarative language. In turn, this makes it easier for the welcome
+page to adopt a style that will more closely resemble the QGIS branding
+seen on the QGIS.org website. 
+
+### Features and Benefits
+
+The revamp will implement a layout mock-up approved by the PSC, which
+will resemble the image below:
+
+![Welcome page layout mockup](images/qep347/welcome-page-layout-mockup.png)
```

Comment:
About the latest projects section, I personnaly prefer seing more items (at least up to five) over large thumbnails, and if possible would love some additional details (exact modification date, QGIS version).